app/assets/stylesheets/tags_input.scss

```diff
@@ -0,0 +1,59 @@
+.tag-words-guide {
+  margin-left: 3rem;
+  font-size: 0.8rem;
+  color: $input-border-color
+}
+
+label.tags-input.active {
+  top: 0.2rem;
+}
+
+div.for-tags-input {
+  margin-top: 2.5rem;
+}
+
+div.tagsinput {
+  // Adjust
+  margin: 0 0 30px 3rem;
+  width: calc(100% - 3rem);
+
+  // jquery tags inut
+  overflow-y: auto;
+
+  //// Materialize
+  // General Styles
+  background-color: transparent;
+  outline: none;
+  height: 3rem;
+  font-size: 1rem;
+}
+
+div.tagsinput input {
+  width: 80px;
+  border: none;
+  outline: none;
+  background: transparent;
+}
+
+@mixin tag-in-input {
+  border-radius: 16px;
+  font-size: 1rem;
+  line-height: 2rem;
+  padding: 0 12px;
+
+  margin-right: 5px;
+  margin-bottom: 5px;
+}
+
+div.tagsinput span.tag {
+  @include tag-in-input;
+  display: block;
+  float: left;
+}
+
+input.not_valid {
+  @include tag-in-input;
+  text-align: center;
+  background: $error-color !important;
+  color: #FFF !important;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`<br>!important should not be used